scripts/generate_sitemap/utils.ts

```diff
@@ -0,0 +1,81 @@
+import { concat, chain } from 'lodash';
+import { SitemapItemLoose } from 'sitemap';
+import urlJoin from 'url-join';
+import regions from '../../src/common/regions';
+import { allCaseStudies } from '../../src/cms-content/learn';
+import articles from '../../src/cms-content/articles';
+import shareImagesData from '../../src/assets/data/share_images_url.json';
+
+const shareImageBaseUrl = shareImagesData.share_image_url;
+
+export function getLocationPageItems(): SitemapItemLoose[] {
+  return chain(regions.all())
+    .map(region => ({
+      url: region.relativeUrl,
+    }))
+    .value();
+}
+
+export function getLearnPageItems(): SitemapItemLoose[] {
+  const topLevelRelativeUrls = [
+    '/learn',
+    '/glossary',
+    '/faq',
+    '/case-studies',
+    '/deep-dives',
+    '/covid-risk-levels-metrics',
+  ];
```

Comment:
Is there a way we can avoid hard-coding these so it doesn't get out-of date?  Else, is there a way we can create "cross-referenced" comments between here and our router / whatever to help us remember to keep these in sync?  Or somehow we can have a test that catches if things get out of sync?

That said, I guess I don't know for sure what happens if we get out-of-sync.  I assume google will still crawl any URL it finds on our site, so maybe it doesn't matter too much.  In that case, maybe we don't even need to bother including everything in the sitemap?  I'm not sure...  Any idea?

Reply:
This may be overkill, but looks like there's an npm package that helps with it: https://github.com/kuflash/react-router-sitemap

Reply:
I investigated `react-router-sitemap` - there is a simpler way, just export the router config for each family of pages and use it to generate the sitemap. This will help us keep the router and sitemap in sync 👌

Reply:
Actually I looked into this a bit more, in order to centralize the config we will need to load all the possible URL params in the router config, which we probably don't want to do.